src/state/slices/opportunitiesSlice/thunks.ts

```diff
@@ -134,6 +134,18 @@ export const fetchAllStakingOpportunitiesUserData = async (
   const { getOpportunitiesUserData, getOpportunityUserData } = opportunitiesApi.endpoints
 
   const userDataPromises = [
+    store.dispatch(
+      getOpportunitiesUserData.initiate(
+        {
+          accountId,
+          defiType: DefiType.Staking,
+          defiProvider: DefiProvider.ThorchainSavers,
+          opportunityType: DefiType.Staking,
+        },
+        // Any previous query without portfolio loaded will be rejected, the first successful one will be cached
```

Comment:
Copied and pasted, remove for all opportunities except FOX Farming in a stacked PR, this was true for FOX farming, not anymore